### ISSUE TITLE
Can't downgrade Xiaomi MJSXJ02CM camera

### ISSUE THREAD

**Issue by HsDanimedia** [telmomarques/xiaomi-360-1080p-hacks#103]
I've tried several SD cards but I can't downgrade to **3.4.2_0062** from **4.0.9_0409**. I've read some article and I know that I can't downgrade my camera using **tf_recovery.img** file. Do you know other options to solve this problem? Thanks!

**Comment by chen2804** [comment]
yes here is the other way to do it,but normally one a few people can be able to do it。 here is the link[https://gist.github.com/belkone/fb63376a9b6b6ebe969e9c8f96c0defb](url)
